src/mountutils.cpp

```diff
@@ -22,4 +24,12 @@ NAN_MODULE_INIT(MountUtilsInit) {
   .ToLocalChecked());
 }
 
+
+void MountUtilsLog(std::string string) {
+  const char* debug = std::getenv("MOUNTUTILS_DEBUG");
```

Comment:
Why not just go with `DEBUG`, which is a bit more conventional? For example, `DEBUG=true` or `DEBUG=mountutils` would be used.